kernel/Dockerfile

```diff
@@ -50,13 +54,24 @@ RUN curl -fsSLO ${KERNEL_SHA256_SUMS} && \
     gpg2 --verify linux-${KERNEL_VERSION}.tar.sign linux-${KERNEL_VERSION}.tar && \
     cat linux-${KERNEL_VERSION}.tar | tar --absolute-names -x && mv /linux-${KERNEL_VERSION} /linux
 
-COPY kernel_config-${KERNEL_SERIES} /linux/arch/x86/configs/x86_64_defconfig
+# When using COPY with more than one source file, the destination must be a directory and end with a /
+COPY kernel_config-${KERNEL_SERIES}* /linux/
 COPY kernel_config.debug /linux/debug_config
 
-RUN if [ -n "${DEBUG}" ]; then \
-    sed -i 's/CONFIG_PANIC_ON_OOPS=y/# CONFIG_PANIC_ON_OOPS is not set/' /linux/arch/x86/configs/x86_64_defconfig; \
-    cat /linux/debug_config >> /linux/arch/x86/configs/x86_64_defconfig; \
-    fi
+RUN case $(uname -m) in \
+    x86_64) \
+        cp /linux/kernel_config-${KERNEL_SERIES} ${KERNEL_DEF_CONF}; \
+        ;; \
+    aarch64) \
+        ${KERNEL_DEF_CONF}=/linux/arch/arm64/configs/defconfig; \
```

Comment:
Shell environment LGTM, I think probably I misunderstood rn's proposal :)  I do think use a shell var will make the code be more consistent in that context.